src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -99,6 +99,11 @@ public MimeMessage prepareEmailMessage(String recipients, String mapNotification
      */
     private MimeMessage prepareEmail(String mapNotificationMessage, String emailSubject,
             String[] recipients) {
+        // Added default sender and subject for email notification
+        if (sender.isEmpty() || subject.isEmpty()) {
```

Comment:
This if statement is not needed since the values will never be empty, they will always have a default set at the top in the @Value field. 

Also the if statement in current state will replace the sender with default if the sender is given by the user but subject is not.